website/docs/r/repository_collaborator.html.markdown

```diff
@@ -48,6 +48,9 @@ resource "github_repository_collaborator" "a_repo_collaborator" {
 The following arguments are supported:
 
 * `repository` - (Required) The GitHub repository
+
+~> Note: The owner of the repository can be passed as part fo the repository name  e.g. `owner-org-name/repo-name`. If owner is not supplied as part of the repository name, can also be supplied by setting the environment variable `GITHUB_OWNER`.
```

Comment:
```suggestion
~> Note: The owner of the repository can be passed as part of the repository name  e.g. `owner-org-name/repo-name`. If owner is not supplied as part of the repository name, it may also be supplied by setting the environment variable `GITHUB_OWNER`.
```

Reply:
Thank you @kfcampbell for the suggestion. Suggestion is commited 🙂